quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusValidator.java

```diff
@@ -256,13 +259,31 @@ private void addDiagnosticsForDuplicates() {
 	}
 
 	private void addDiagnosticsForMissingRequired(PropertiesModel document) {
+		BufferedWriter writer = null;

```

Comment:
I think QuarkusValidator should be not updated?

Reply:
oopss, that should have never been commited. Fixed